.github/workflows/integration-test-enterprise.yaml

```diff
@@ -46,8 +46,9 @@ jobs:
         - '2.7.2.0'
         - '2.8.4.0'
         - '3.0.2.0'
-        - '3.1.1.3'
-        - '3.2.2.1'
+        - '3.1.1.4'
+        - '3.2.2.3'
+        - '3.3.0.0'
```

Comment:
is there a reason the series tags (e.g. `kong/kong-gateway:3.1`) ?

Reply:
Good point. I've changed it to use major.minor. PTAL